On Debian 'sid' systems there is no linux/dirent.h header when
linux-libc-headers 2.6.28 are installed.

---
 linux-user/syscall.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- qemu-0.9.1.orig/linux-user/syscall.c
+++ qemu-0.9.1/linux-user/syscall.c
@@ -74,11 +74,11 @@
 #include <linux/unistd.h>
 #include <linux/utsname.h>
 #include <linux/cdrom.h>
 #include <linux/hdreg.h>
 #include <linux/soundcard.h>
-#include <linux/dirent.h>
+#include <dirent.h>
 #include <linux/kd.h>
 
 #include "qemu.h"
 #include "qemu_spinlock.h"
 
